bitnami/nginx/1.24/debian-11/Dockerfile

```diff
@@ -23,7 +23,7 @@ ENV HOME="/" \
 COPY prebuildfs /
 SHELL ["/bin/bash", "-o", "errexit", "-o", "nounset", "-o", "pipefail", "-c"]
 # Install required system packages and dependencies
-RUN install_packages ca-certificates curl libcrypt1 libgeoip1 libpcre3 libssl1.1 openssl procps zlib1g
+RUN install_packages ca-certificates curl libcrypt1 libgeoip1 libpcre3 libssl1.1 openssl procps zlib1g gettext-base
```

Comment:
Please use `gettext` as in other images like [appsmith](https://github.com/bitnami/containers/blob/main/bitnami/appsmith/1/debian-12/Dockerfile#L26)